package.json

```diff
@@ -75,6 +75,7 @@
     "jest-environment-jsdom": "29.7.0",
     "nodemon": "3.0.1",
     "prettier": "3.0.3",
+    "qlik-chart-modules": "0.59.0",
```

Comment:
if this pkg is provided by both SNT and PVT, it's better to exclude it from nebula-table-utils bundle by adding it in the rollup.config.js file in externals at here: https://github.com/qlik-oss/nebula-table-utils/blob/e411cfca185820bb641a7ec3c0f91dbb9e0c4ff2/rollup.config.js#L19-L28